src/app/clusters/icd-management-server/icd-management-server.cpp

```diff
@@ -69,6 +73,13 @@ CHIP_ERROR IcdManagementAttributeAccess::Read(const ConcreteReadAttributePath &
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR IcdManagementAttributeAccess::ReadIdleModeInterval(EndpointId endpoint, AttributeValueEncoder & encoder)
+{
+    uint32_t idle_interval = CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL;
+    ReturnErrorOnFailure(encoder.Encode(idle_interval));
+    return CHIP_NO_ERROR;
+}
```

Comment:
It is not possible to use the generation configuration instead of adding a new configuration define?

Reply:
I modify the code to get the values from the generated config, but we still need a fall-back mechanism, so I have also added the default configuration for each attribute, with the exception of the counter, which defaults to zero.

Reply:
> but we still need a fall-back mechanism

Why?

Reply:
Because the default values are obtained from the attributes store, which may return an error.

Reply:
Oh, I see, you want a fall-back for the attribute store, not a fall-back for compile-time config....